docker-compose.yaml

```diff
@@ -35,24 +33,11 @@ services:
   ffc-sfd-data-redis:
     image: redis:6.2.7
   
-  cosmos-db:
-    container_name: cosmos-db
+  ffc-sfd-data-cosmos-db:
+    container_name: ffc-sfd-data-cosmos-db
```

Comment:
Do we need to set container name?   What's the value fixing it to this?

Also as it's already set in the main compose file and this isn't changing it, doesn't need to be here at all

Reply:
removed